addon/services/cloudinary.js

```diff
@@ -181,5 +181,92 @@ export default Ember.Service.extend({
     }
 
     return `${protocol}${subDomain}.${domain}${pathname}`;
+  },
+
+
+  /** @property {String[]} _concatenatedTransforms */
+  _concatenatedTransforms: computed('config.CONCATENATED_TRANSFORMS', function() {
+    // Protection from Strings
+    const concatenatedTransforms = this.get('config.CONCATENATED_TRANSFORMS') || [];
+    return Array.isArray(concatenatedTransforms) ? concatenatedTransforms : [concatenatedTransforms];
+  }),
+
+  /**
+   * @method getURL
+   * @param  {String}          [publicId]           Public id of image in Cloudinary
```

Comment:
You can omit the brackets since `publicId` should become just a string instead of array